MAUI/DataForm/Validation.md

```diff
@@ -49,6 +51,97 @@ public DateTime JoinDate { get; set; }
 
 Refer [here](https://help.syncfusion.com/maui/dataform/data-annotations#dateformdaterange-attribute) to know more about date range attribute in DataForm.
 
+### Validate using the IDataErrorInfo interface
+
+You can validate the data by implementing the [IDataErrorInfo](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.idataerrorinfo?view=net-6.0) interface in the data object class.
+
+{% tabs %}
+{% highlight C# %}
+
+public class EmployeeInfo : IDataErrorInfo
+{
+    public string EmployeeID { get; set; }
+
+    public string Name { get; set; }
+
+    public string Title { get; set; }
+
+    [Display(AutoGenerateField = false)]
+    public string Error
+    {
+        get
+        {
+            return string.Empty;
+        }
+    }
+
+    [Display(AutoGenerateField = false)]
+    public string this[string columnName]
+    {
+        get
+        {
+            if (columnName == nameof(Title) && this.Title.Contains("Marketing"))
+            {
+                return "Marketing is not allowed";
+            }
+
+            return string.Empty;
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+You can download the entire source code [here](https://github.com/SyncfusionExamples/maui-dataform/tree/master/DataErrorInfoSample)
```

Comment:
check sample reference content and follow same pattern.
https://help.syncfusion.com/maui/dataform/grouping#changing-the-padding-for-group-headers-and-editors

Reply:
pattern updated